### PR TITLE
[TW-1567] Doc Private network read/write separation for new enterprise packaging

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Updated the Private API Network and Admin pages to reflect the read/write separation, replaced a screenshot

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md [hunk 1]
```diff
@@ -38,11 +38,13 @@ contextual_links:
...
+
 Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources.
 
 [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them.
 
-<img alt="Private API Network overview" src="https://assets.postman.com/postman-docs/v10/private-api-network-overview-v10-4.jpg"/>
+<img alt="Private API Network overview" src="https://assets.postman.com/postman-docs/v10/private-api-network-permissions-v10.jpg"/>
```
Comment (HimanshuS129) [inline src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:47]: I think with this screenshot we intent to show the Folder Manager thing, but this page doesn't mention API Network folder managers. This API Network folder manager is sort of an internal construct limited to API Network's boundaries and hence not visible in Team settings.
A screenshot like this should be better
Screenshot 2023-12-06 at 11 57 53 AM

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 2]
```diff
@@ -270,8 +270,8 @@ You can add team members to the [Internal API Management solution](https://www.p
 When you add team members to the Internal API Management solution:
 
 * You can then assign them the [API Network Manager and API Governance Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) roles. Learn more about [managing team roles](#managing-team-roles).
-* All team members get access to the Private API Network.
-* Only team members added to this solution can be assigned the [API Editor](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) role in Postman APIs, enabling them to create and edit an unlimited amount of APIs. Team members not added to this solution will only have the [API Viewer](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) role.
+* Only the team members with a seat can publish/request to publish changes to the Private API Network, while the rest of the team can consume/read the network.
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:273]:
```suggestion
* Only the team members with a seat can publish or request to publish changes to the Private API Network, while the rest of the team can consume and read from the network.
```

Comment (jkonrath-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md:273]: I think "a seat" needs to be clarified. (Maybe "added to this solution" ?)

**Review by jkonrath-postman — APPROVED** [review]
Two little things but LGTM!

**Pull request merged**